pom.xml

```diff
@@ -127,6 +127,12 @@
             <classifier>classes</classifier>
         </dependency>
 
+        <dependency>
+            <groupId>ca.uhn.hapi.fhir</groupId>
+            <artifactId>hapi-fhir-opensrp-security-config</artifactId>
+            <version>5.4.0-PRE5-SNAPSHOT</version>
```

Comment:
The version for this should be start from `0.0.1`

Reply:
Done, here is the new [repo](https://github.com/opensrp/hapi-fhir-keycloak/blob/master/pom.xml).